src/Windows/Avalonia.Direct2D1/Media/DrawingContextImpl.cs

```diff
@@ -265,14 +268,17 @@ public void DrawText(IBrush foreground, Point origin, IFormattedTextImpl text)
             {
                 var impl = (FormattedTextImpl)text;
 
+                impl.ApplyDrawingEffects(this);
```

Comment:
Why are the drawing effects applied and removed each time the text is rendered? Why not just apply them when the `FormattedTextImpl` is created?